ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/Util/TestBlock.hs

```diff
@@ -267,6 +271,19 @@ data instance CodecConfig TestBlock = TestCodecConfig
 instance BlockHasCodecConfig TestBlock where
   getCodecConfig = const TestCodecConfig
 
+data Unversioned = Unversioned
+  deriving (Eq, Show)
+
+instance HasNetworkProtocolVersion TestBlock where
+    type NodeToNodeVersion TestBlock   = Unversioned
+    type NodeToClientVersion TestBlock = Unversioned
+    supportedNodeToNodeVersions   _ = Unversioned :| []
+    supportedNodeToClientVersions _ = Unversioned :| []
+    mostRecentNodeToNodeVersion   _ = Unversioned
+    mostRecentNodeToClientVersion _ = Unversioned
+    nodeToNodeProtocolVersion     _ Unversioned = NtN.NodeToNodeV_1
+    nodeToClientProtocolVersion   _ Unversioned = NtC.NodeToClientV_2
+
```

Comment:
We have a default instance that you can use, so replace this with:
```haskell
instance HasNetworkProtocolVersion TestBlock where
```

Reply:
ok